src/styles/main.scss

```diff
@@ -0,0 +1,42 @@
+@import "keyframes";
+
+$borderRadius: 8px;
+$handWidth: 100px;
+
+.stopwatch {
```

Comment:
separate file per bem block

Reply:
<img width="652" alt="image" src="https://user-images.githubusercontent.com/102411130/231977267-5de3aaef-4a67-4bd8-95f2-524e5c787751.png">

Reply:
still not fixed